front-end/src/app/shared/services/login.service.ts

```diff
@@ -71,11 +77,6 @@ export class LoginService extends DestroyerComponent {
     this.cookieService.delete(environment.ffapiSecurityConsentCookieName);
   }
 
-  public clearUserLoggedInCookies() {
-    this.clearUserFecfileApiCookies();
-    this.cookieService.delete(environment.sessionIdCookieName);
```

Comment:
This didn't work.  Apparently, the front-end is incapable of clearing cookies with the httpOnly flag set to true.